src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -122,6 +124,29 @@
     </ItemGroup>
 
     <Copy Condition="'$(BUILDTOOLS_OVERRIDE_RUNTIME)' != ''" SourceFiles="@(TestRuntimeSource)" DestinationFolder="$(TestPath)%(TestTargetFramework.Folder)" />
+
+    <!-- the helix test infrastructure uses the generated assembly list to determine the runtime binaries to restore -->
+    <Message Text="Generating $(OutDir)AssemblyList.txt" />
```

Comment:
This is a little odd in this location, if I'm reading this correctly, because it's in the "CopyTestToTestDirectory" target... which is something we wouldn't want to do for tests that are about to be zipped/uploaded, since that will contain all the product binaries inside the test package.